plugin/src/main/java/at/helpch/chatchat/util/Kyorifier.java

```diff
@@ -0,0 +1,75 @@
+package at.helpch.chatchat.util;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Stack;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import net.kyori.adventure.text.format.NamedTextColor;
+import net.kyori.adventure.text.format.TextDecoration;
+
+public final class Kyorifier {
+    private static final Map<Character, String> COLOURS = new HashMap<>(15);
+    private static final Map<Character, String> FORMATTERS = new HashMap<>(6);
+    private static final Pattern pattern = Pattern.compile("&(?<code>[\\da-fk-or])|[&{\\[<]?[#x](?<hex>(&?[A-Fa-f\\d]){6})[}\\]>]?");
+
+    static {
```

Comment:
maybe a nitpick as im personally not a fan of using the static block. id prefer this to be change to use googles [ImmutableMap Builder](https://javadoc.io/doc/com.google.guava/guava/latest/com/google/common/collect/ImmutableMap.Builder.html) instead. (or just Map.of() for formatters map ,Map.of() only supports up to 10 entries i think)

will see what others think instead however

Reply:
sure. can change this as well. will do it tomorrow most likely.